Mathlib/Analysis/Normed/Algebra/Basic.lean

```diff
@@ -50,3 +50,15 @@ instance [ProperSpace 𝕜] : CompactSpace (characterSpace 𝕜 A) := by
 end CharacterSpace
 
 end WeakDual
+
+section cobounded
+
+open Filter Bornology
+
+@[simp]
+theorem tendsto_algebraMap_cobounded (𝕜 𝕜' : Type*) [NormedField 𝕜] [SeminormedRing 𝕜']
+    [NormedAlgebra 𝕜 𝕜'] [NormOneClass 𝕜'] :
+    Tendsto (algebraMap 𝕜 𝕜') (cobounded 𝕜) (cobounded 𝕜') :=
+  algebraMap_cobounded_le_cobounded 𝕜 𝕜'
```

Comment:
Can you put this in place of `algebraMap_cobounded_le_cobounded`, and then deprecate that declaration? They're the same up to defeq and the pre-existing version in Mathlib just has less API, so there's no reason to keep it.